<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgVodaCtrl</class>
 <widget class="QDialog" name="dlgVodaCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Завдання</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Мін. температура</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="sb_Tv_min">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>150.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Макс. температура</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="sb_Tv_max">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>150.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Мін. рівень</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Макс, рівень</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="sb_Lv_min">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="singleStep">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="sb_Lv_max">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="singleStep">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Управління</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Режим</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_AM_Tv">
        <item>
         <property name="text">
          <string>РУЧ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>АВТ</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>306</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="bnExit">
     <property name="text">
      <string>Закрити</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>99</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>250</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame
{
	background-image: url(:/mnemo/voda.svg);
	background-repeat: no-repeat;

}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLineEdit" name="le_Lvoda">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>87</y>
           <width>45</width>
           <height>24</height>
          </rect>
         </property>
        </widget>
        <widget class="QSlider" name="sl_Lv_min">
         <property name="geometry">
          <rect>
           <x>122</x>
           <y>114</y>
           <width>20</width>
           <height>119</height>
          </rect>
         </property>
         <property name="whatsThis">
          <string>Постійне завдання на перший параметер</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider::groove:vertical {
     background: rgba(0, 0, 0, 0);
/*     position: absolute;  абсолютная позиция в 4px слева и справа от виджета. установка полей виджета также будет работать... */
/*     left: 4px; right: 4px;*/
 }

 QSlider::handle:vertical {
	
	image: url(:/slider/pict/SP_1.png);
 }
</string>
         </property>
         <property name="maximum">
          <number>4000</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>40</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="sliderPosition">
          <number>0</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
        <widget class="QSlider" name="sl_Lv_max">
         <property name="geometry">
          <rect>
           <x>87</x>
           <y>114</y>
           <width>20</width>
           <height>119</height>
          </rect>
         </property>
         <property name="whatsThis">
          <string>Розрахункове завдання</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider::groove:vertical {
     background: rgba(0, 0, 0, 0);
/*     position: absolute;  абсолютная позиция в 4px слева и справа от виджета. установка полей виджета также будет работать... */
/*     left: 4px; right: 4px;*/
 }

 QSlider::handle:vertical {
	image: url(:/slider/pict/SPR_1.png);
 }
</string>
         </property>
         <property name="maximum">
          <number>4000</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>80</number>
         </property>
         <property name="value">
          <number>2500</number>
         </property>
         <property name="sliderPosition">
          <number>2500</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
        <widget class="QProgressBar" name="pb_Lvoda">
         <property name="geometry">
          <rect>
           <x>104</x>
           <y>118</y>
           <width>21</width>
           <height>111</height>
          </rect>
         </property>
         <property name="maximum">
          <number>4000</number>
         </property>
         <property name="value">
          <number>500</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_Tvoda">
         <property name="geometry">
          <rect>
           <x>27</x>
           <y>171</y>
           <width>45</width>
           <height>24</height>
          </rect>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_Y_08">
         <property name="geometry">
          <rect>
           <x>73</x>
           <y>49</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
min-width: 16px;
min-height: 16px;
max-width: 16px;
max-height: 16px;
}
QCheckBox:checked
{
	image: url(:/valves/lamp_green_16x16.png);
}

QCheckBox:unchecked
{
	image: url(:/valves/lamp_off_16x16.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 16px;
	width: 16px;
}
</string>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_Y_09">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>49</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
min-width: 16px;
min-height: 16px;
max-width: 16px;
max-height: 16px;
}
QCheckBox:checked
{
	image: url(:/valves/lamp_green_16x16.png);
}

QCheckBox:unchecked
{
	image: url(:/valves/lamp_off_16x16.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 16px;
	width: 16px;
}
</string>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_X_07">
         <property name="geometry">
          <rect>
           <x>87</x>
           <y>41</y>
           <width>20</width>
           <height>32</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
min-width: 20px;
min-height: 32px;
max-width: 20px;
max-height: 32px;
}
QCheckBox:checked
{
	
	
	image: url(:/valves/valve_green_l_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/valves/valve_off_l_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_X_08">
         <property name="geometry">
          <rect>
           <x>121</x>
           <y>41</y>
           <width>20</width>
           <height>32</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
min-width: 20px;
min-height: 32px;
max-width: 20px;
max-height: 32px;
}
QCheckBox:checked
{
	
	
	image: url(:/valves/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/valves/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>98</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="text.qrc"/>
  <include location="../../../pict/lib/valves.qrc"/>
  <include location="../../../lib/picture.qrc"/>
  <include location="mnemo/mnemo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bnExit</sender>
   <signal>clicked()</signal>
   <receiver>dlgVodaCtrl</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
